src/officehours_api/tests.py

```diff
@@ -163,3 +165,83 @@ def test_meeting_start_notifies_attendees(self, mock_twilio: mock.MagicMock):
             receivers >=
             {'+15555550000', '+15555550001'}
         )
+
+
+class MeetingSerializerTestCase(TestCase):
+    """
+    Test the scenarios where the backend type is invalid, valid,
+    patched to none, and patched to invalid.
+    """
+
+    def setUp(self):
+        self.user1 = User.objects.create(username='user1')
+        self.user2 = User.objects.create(username='user2')
+        self.inperson_queue = Queue.objects.create(name='test queue', status='open', allowed_backends=['inperson'])
+        self.zoom_queue = Queue.objects.create(name='test queue', status='open', allowed_backends=['zoom'])
+        self.inperson_queue.hosts.add(self.user1)
+        self.zoom_queue.hosts.add(self.user1)
+
+    def test_backend_type_invalid(self):
+        data = {
+            'queue': self.zoom_queue.id,
+            'attendee_ids': [self.user2.id],
+            'agenda': 'test agenda',
+            'assignee_id': self.user1.id,
+            'backend_type': 'inperson'
+        }
+        serializer = MeetingSerializer(data=data)
+        with self.assertRaises(ValidationError) as cm:
+            serializer.is_valid(raise_exception=True)
+        error = str(cm.exception.detail['non_field_errors'][0])
+        self.assertEqual(error, "inperson is not one of the queue's allowed backend types (['zoom'])")
+
+    def test_backend_type_valid(self):
+        data = {
+            'queue': self.inperson_queue.id,
+            'attendee_ids': [self.user2.id],
+            'agenda': 'test agenda',
+            'assignee_id': self.user1.id,
+            'backend_type': 'inperson'
+        }
+        serializer = MeetingSerializer(data=data)
+        valid = serializer.is_valid(raise_exception=False)
+        self.assertTrue(valid)
+
+    def test_patch_backend_type_none(self):
```

Comment:
In line with the other comment, I think this makes it a bit clearer.
```suggestion
    def test_patch_without_backend_type(self):
```